app/full_text_search.py

```diff
@@ -125,6 +150,11 @@ def search(repo, query_txt, max_results):
     # Remove double quotes so that we can safely apply enclose_in_double_quotes().
     query_txt = re.sub('"', '', query_txt)
     romanized_query = create_query_txt(query_txt)
+    kanji_words_in_query_txt = script_variant.find_kanji_word(query_txt)
```

Comment:
If you want to apply the logic for non-kanji words as well, you should extract all words here instead of kanji words.